docs/api/system-preferences.md

```diff
@@ -274,3 +282,51 @@ Returns `Boolean` - `true` if an inverted color scheme, such as a high contrast
 theme, is active, `false` otherwise.
 
 [windows-colors]:https://msdn.microsoft.com/en-us/library/windows/desktop/ms724371(v=vs.85).aspx
+
+### `systemPreferences.getEffectiveAppearance()` _macOS_
+
+Returns `String` - Can be `dark`, `light` or `unknown`.
+
+Gets the macOS appearance setting that is currently applied to your application,
+maps to [NSApplication.effectiverAppearance](https://developer.apple.com/documentation/appkit/nsapplication/2967171-effectiveappearance?language=objc)
```

Comment:
[NSApplication.effectiveAppearance]?

Reply:
This isn't objective c, it's intended to indicate a property style value to JS devs that they can click on to learn more

Reply:
Adding an r into "effectiveAppearance" so it's "effectiverAppearance" does that?

Reply:
It's more effective that way

Reply:
*more effectiver